index.test.js

```diff
@@ -0,0 +1,33 @@
+const updog = require('./index.js');
+
+test('Test truthiness of updog', () => {
+  expect(updog()).toBeTruthy();
+});
+
+test('Test that updog is not divisible', () => {

```

Comment:
You should make sure you can't multiply it either, just to be safe

Reply:
I'll update this ASAP. Thanks for the input!

Reply:
Done, I made some additional additions as well as that thing you said:
Added the requested test case and another.
Set up eslint to check the source against the standard js style guide.
Reformatted to conform to the style guide.
Added JSDoc style documentation.

I have a good feeling that we are getting close to a 1.0.0 official release! We should start talking soon about publishing to npm.

Reply:
Great! I love how the developers on this package go above and beyond for their consumers